lib/Admin/tiles/image.php

```diff
@@ -13,27 +13,19 @@
 $image_id        = $item['id'] ?? '';
 $image_filename  = $item['filename'] ?? '';
 $original_url    = $item['embeds']['original']['url'] ?? '';
-$thumbnail_url   = $item['embeds']['ThumbnailPNG']['url'] ?? '';
-$skeleton_url    = $item['embeds']['SkeletonPNG']['url'] ?? '';
-$pager_url       = $item['embeds']['PagerPNG']['url'] ?? '';
+$templated_url   = $item['embeds']['templated']['url'] ?? '';
```

Comment:
Discovered that there was a new templated url in the API that we could leverage to generate the various sizes. This allows for less configuration in Widen, removing the previous requirement of needing specific image sizes created in Widen.

Reply:
Nice find Joe